training/src/controllers/requests/handlers/getRequest.handler.ts

```diff
@@ -0,0 +1,10 @@
+import { RequestContext } from '@mikro-orm/core';
+import { Request } from '../../../entities/request.entity';
+
+export const getRequest = async (id: string) => {
+  const em = RequestContext.getEntityManager();
+
+  const request = await em.findOneOrFail(Request, { id });
+  await em.populate(request, ['tender']);
```

Comment:
In simple cases populate can just be included in the find, no need for the extra statement